packages/manager/modules/manager-components/src/resource-selector/template.html

```diff
@@ -0,0 +1,11 @@
+<div class="oui-tile d-flex m-2">
+    <h6 class="w-50" data-ng-bind="$ctrl.title"></h6>
+    <div class="text-right ml-auto mr-3" ng-transclude="price">
+        <h6 class="m-0">
+            <span data-ng-bind="$ctrl.price"></span>
```

Comment:
does this info shown if resourceSelectorPrice is provided?

Reply:
No... The user can override it using the `resourceSelectorPrice`. The user can directly give the price string. But `resourceSelectorPrice` takes higher precedence